docs/sources/reference/playlist.md

```diff
@@ -15,54 +15,54 @@ The Playlist is a special type of Dashboard that rotates through a list of Dashb
+Once all the wanted dashboards are added to a playlist, you can save this playlist by clicking on the green **Save** button. This will generate a unique URL for you playlist which can be shared if needed. Click on the generated URL or on the **Play** button from the **Saved playlists** list to start the playlist. If you want to share the URL, right click on the URL and copy the URL link and share.
 
-## Starting the playlist
+## Start a playlist
 
-Also, if you want, you can start the playlist without saving it by clicking on the green "Start" button at the bottom.
+Also, if you want, you can start the playlist without saving it by clicking on the green **Start** button at the bottom.
 
-## Controlling the Playlist
+## Control a playlist
 
 Playlists can also be manually controlled utilizing the Playlist controls at the top of screen when in Playlist mode.
 
 Click the stop button to stop the Playlist, and exit to the current Dashboard.
 Click the next button to advance to the next Dashboard in the Playlist.
 Click the back button to rewind to the previous Dashboard in the Playlist.
 
-## TV or Kiosk Mode
+## TV or kiosk mode
 
-In TV mode the top navbar, row & panel controls will all fade to transparent.
+In TV mode the top navbar, row and panel controls will all fade to transparent.
 
 This happens automatically after one minute of user inactivity but can also be toggled manually
 with the `d v` sequence shortcut, or by appending the parameter `?inactive` to the dashboard URL. Any mouse movement or keyboard action will
-restore navbar & controls.
+restore navbar and controls.
 
 Another feature is the kiosk mode - in kiosk mode the navbar is completely hidden/removed from view. This can be enabled with the `d k`
 shortcut.
 
 To put a playlist into kiosk mode, use the `d k` shortcut after the playlist has started. The same shortcut will toggle the playlist out of kiosk mode.
 
-### Linking to the Playlist in Kiosk Mode
+### Link to a playlist in kiosk mode
 
-If you want to create a link to the playlist with kiosk mode enabled:
+To create a link to the playlist with kiosk mode enabled:
 
-1. Copy the Start Url (by right clicking on the Play button and choosing Copy link address).
+1. Copy the Start Url (right click  the **Play** button and choose **Copy link address**).
 2. Add the `?kiosk` parameter to the url.
 
-For example, to open the first playlist on the Grafana Play site in kiosk mode: [http://play.grafana.org/playlists/play/1?kiosk](http://play.grafana.org/playlists/play/1?kiosk)
+View the first playlist on the Grafana Play site in kiosk mode here: [http://play.grafana.org/playlists/play/1?kiosk](http://play.grafana.org/playlists/play/1?kiosk).
```

Comment:
This is an example of the second item, which is not clear right now (not sure it was clear before). Should this be one the same line as 2. ?

Reply:
Looking more closely, that example applies to both steps 1 and 2, and refers to a general way to create a link to any playlist with kiosk mode enabled. No need to nest it under 2, so I'd actually correct the whole section to something like this:

To create a link to a playlist with kiosk mode enabled:

1. Copy the Start URL (right click on the **Play** button, then choose **Copy link address**).
2. Add the `?kiosk` parameter to the url.

For example, to open the first playlist on the Grafana Play site in kiosk mode, the URL should look like this: 
http://play.grafana.org/playlists/play/1?kiosk